README.md

```diff
@@ -34,7 +34,11 @@ The CI for this repository is written using GitHub Actions. Here is a list of cu
 
 All scripts used in the CI checks are located in the `scripts` folder.
 
-To avoid accidently pushing information about your private gem, configure the `pre-push` git hook by running `git config core.hooksPath .githooks` in your terminal. This git hook will run most of the CI checks and block the push if you don't pass all the tests.
+To avoid pushing a PR with errors, configure the `pre-push` git hook by running:
+```shell
+$ git config core.hooksPath .githooks
+```
+This git hook will run most of the CI checks and block the push if you don't pass all the tests.
```

Comment:
~~~suggestion
To avoid pushing a PR with errors, configure the `pre-push` git hook by running:

```shell
$ git config core.hooksPath .githooks
```

This git hook will run most of the CI checks and block the push if you don't pass all the tests.
~~~